cmake/BLTOptions.cmake

```diff
@@ -58,6 +58,10 @@ mark_as_advanced(ENABLE_CLANG_CUDA)
 set(BLT_CLANG_CUDA_ARCH "sm_30" CACHE STRING "Compute architecture to use when generating CUDA code with Clang")
 mark_as_advanced(BLT_CLANG_CUDA_ARCH)
 option(ENABLE_HIP         "Enable HIP support" OFF)
+option(ENABLE_CLANG_HIP   "Enable Clang's native HIP support" OFF)
+mark_as_advanced(ENABLE_CLANG_HIP)
+set(BLT_CLANG_HIP_ARCH "gfx906" CACHE STRING "AMDGPU ISA target to use when generating HIP code with Clang" )
```

Comment:
Is this a sensible default?

Reply:
Yes, that is what is supported on both corona and rznevada.

Reply:
`rznevada` is actually gfx908 (MI100). Corona and rzwhamo are/were gfx906.

Either one would be a sensible default, or we could specify both `gfx908;gfx906` and generate a fat-binary. Code for each device has to be linked separately, though, so link times would increase.

Reply:
`gfx906` covers both the AMD MI50 and MI60, which are still the most common in the field. I think that's the most sensible choice for the moment and there will be host-configs for most of the early MI100 platforms. It may make sense to change the default platform at a later time.